src/tools/image/FileOpen.java

```diff
@@ -6,7 +6,6 @@
 import javax.swing.JOptionPane;
 
 public class FileOpen {
-  private final String ERR_MSG_1 = "Warning! Could not load the file!";
```

Comment:
For this one no, we have in-line error messages all over the place in the rest of the code. The extraction to a constant does not buy us anything.